mobile/src/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {memo} from 'react';
+import {Button as PaperButton} from 'react-native-paper';
+import {components} from '../../../config/theme';
+import {styles} from './styles';
+import {StyledButtonProps} from './types';
+import {ButtonStyleDictionary} from './const';
+
+const Button = ({style, children, type, ...props}: StyledButtonProps) => (
+  <PaperButton
+    style={[
+      type
+        ? ButtonStyleDictionary[type].style
```

Comment:
To jeden komentarz ale dotyczy wszystkich buttonów, fajne jak różne typy buttonów będą zaszyte w theme a nie w warunkowych stylach w UI.